src/tools/allocator-recipe-resolver.ts

```diff
@@ -209,3 +207,25 @@ export function findLongRunningArcId(recipe: Recipe): string | null {
   const arcIdAnnotation = recipe.getAnnotation('arcId');
   return arcIdAnnotation ? Object.values(arcIdAnnotation.params)[0].toString() : null;
 }
+
+/** Intermediary in the recipe resolution process that holds stores. */
+class ManifestResolutionAdapter implements ResolutionContext {
+
+  constructor(public readonly globalInfo: Manifest) {}
+
+  findStoreById(id: string): AbstractStore | null {
+    return null;
+  }
+
+  findStoresByType<T extends Type>(type: T, options?: { tags: string[], subtype?: boolean }): AbstractStore[] {
+    return [];
+  }
```

Comment:
these should have a non-empty implementation for Manifest; the thing that should be null for manifest is globalInfo.